libcontainer/init_linux.go

```diff
@@ -412,7 +413,7 @@ func fixStdioPermissions(config *initConfig, u *user.ExecUser) error {
 	} {
 		var s unix.Stat_t
 		if err := unix.Fstat(int(fd), &s); err != nil {
-			return err
+			return &os.PathError{Op: "fstat", Path: "fd " + strconv.Itoa(int(fd)), Err: err}
```

Comment:
Probably mostly a nit, but wondering, given that `fd` is a `uintptr`, if `FormatUint()` would be a closer fit?

```suggestion
			return &os.PathError{Op: "fstat", Path: "fd " + strconv.FormatUint(uint64(fd), 10), Err: err}
```

Reply:
🤔 I should propose a `strconv.UItoa()` function in Golang 🤔

Reply:
> Probably mostly a nit, but wondering, given that `fd` is a `uintptr`, if `FormatUint()` would be a closer fit?

It still needs a typecast, and adds additional argument (`10`). I like Atoi because it's a tad shorter.

The difference between FormatInt and FormatUint is latter do not check if the argument is negative.

All in all, it's all the same here, I don't see how one way is better than the other.

I'm even fine with `fmt.Sprintf("fd %d", fs)` -- it is slower but we don't care since it's an error path.

Reply:
>  I like Atoi because it's a tad shorter.

Yes, I wish there was a `strconv.UItoa()`

> All in all, it's all the same here, I don't see how one way is better than the other.

It was mostly a nit; wondering if `gosec` will complain about it once we enabled it 🤔